doc/nrf/known_issues.rst

```diff
@@ -552,6 +552,14 @@ Front-End Modules API implementation missing
 SoftDevice Controller
 =====================
 
+.. rst-class:: v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0
+
+DRGN-11963: LL control procedures cannot be initiated at the same time
+  The LL control procedures LE start encryption and LE connection parameter update cannot be initiated at the same time:
```

Comment:
@rugeGerritsen It affects 2x LE start encryption as well, update text to same "... at the same time or more than once"?

Reply:
@greg-fer Do you agree with this sentence change?

Reply:
Yes.